docs/design/generics/overview.md

```diff
@@ -329,20 +360,23 @@ fn PrintIt(p: Song*) {
 }
 ```
 
-Inside the function body, you can treat the generic type parameter just like any
-other type. There is no need to refer to or access generic parameters
-differently because they are defined as generic, as long as you only refer to
-the names defined by [type-of-type](#type-of-types) for the type parameter.
+Inside the function body, you can treat the facet parameter just like any other
+type. There is no need to refer to or access generic parameters differently
+because they are defined as generic, as long as you only refer to the names
+defined by [facet type](#facet-types) for the facet parameter.
 
-You may also refer to any of the methods of interfaces required by the
-type-of-type using a
-[qualified member access expression](#accessing-members-of-interfaces), as shown
-in the following sections.
+You may also refer to any of the methods of interfaces required by the facet
+type using a
+[qualified member access expression](#accessing-members-of-interfaces).
 
-A function can have a mix of generic, template, and regular parameters.
-Likewise, it's allowed to pass a template or generic value to a generic or
-regular parameter. _However, passing a generic value to a template parameter is
-future work._
+A function can have a mix of checked, template, and regular parameters. A
+checked parameter is defined using a symbolic binding pattern, a template
+parameter using a template binding pattern, and a regular parameter using a
+runtime binding pattern. Likewise, it's allowed to pass a symbolic or template
```

Comment:
```suggestion
A function can have a mix of checked, template, and regular parameters. A
checked parameter is defined using a symbolic binding pattern, a template
parameter is defined using a template binding pattern, and a regular parameter is defined using a
runtime binding pattern. Likewise, it's allowed to pass a symbolic or template
```

I originally read this as

> A checked parameter is defined using (a symbolic binding pattern, a template parameter using a template binding pattern, and a regular parameter using a runtime binding pattern).

Alternatively, I think something like this could work:

> Each kind of parameter is defined using a different syntax: a checked parameter is uses a symbolic binding pattern, a template parameter uses a template binding pattern, and a regular parameter uses a runtime binding pattern.

Reply:
I used the alternative, but also noticed I needed to update the next sentence.